atm.js

```diff
@@ -1,21 +1,62 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
+$( document ).ready( function(){
+
+var cBal = 0
+var sBal = 0
+
+//Had issues making this work
+// $("div.zero").css({
+//       "background-color": "000000",
+//     });
+
+//Checking Account - deposit and withdraw buttons
+  //When user enters a value and clicks the deposit button, update account balance
+    //User enters value in cUserInput field
+    //user clicks cDepositButton - update checkingBalance to reflect user deposit
+    //user clicks cWithdrawButton - update checkingBalance to reflect user withdraw
+    //if userinput withdraw makes checkingBalance go below zero, do not allow
+
+//Checking Account Deposit
+    $( "#cDepositButton" ).on("click", function(){
+      var cDepositUserInput = parseInt($( "#cUserInput" ).val());
+      cBal = cBal + cDepositUserInput;
+      $( "#checkingBalance" ).text( "$" + cBal );
+    })
+
+//Checking Account Withdraw
+$( "#cWithdrawButton" ).on("click", function(){
+  var cWithdrawUserInput = parseInt($( "#cUserInput" ).val());
+  if(cBal >= cWithdrawUserInput) {
+  cBal = cBal - cWithdrawUserInput;
+    $( "#checkingBalance" ).text( "$" + cBal );
+  }
+  else if(cBal < cWithdrawUserInput) {
+    var answer = confirm("Insufficient Funds!")
+  }
+})
+
+//Savings Account - deposit and withdraw buttons
+  //When user enters a value and clicks the deposit button, update account balance
+    //User enters value in sUserInput field
+    //user clicks sDepositButton - update savingsBalance to reflect user deposit
+    //user clicks sWithdrawButton - update savingsBalance to reflect user withdraw
+
+//Savings Account Deposit
+$( "#sDepositButton" ).on("click", function(){
+  var sUserInput = parseInt($( "#sUserInput" ).val());
+  sBal = sBal + sUserInput;
+  $( "#savingsBalance" ).text( "$" + sBal );
+})
+
+//Savings Account Withdraw
+$( "#sWithdrawButton" ).on("click", function(){
+  var sWithdrawUserInput = parseInt($( "#sUserInput" ).val());
+  if(sBal >= sWithdrawUserInput) {
+  sBal = sBal - sWithdrawUserInput;
```

Comment:
this code looks very similar to the code handling the checking account withdrawals. Can you think of a way to combine the functionality between both of these accounts?